CONTRIBUTING.md

```diff
@@ -186,7 +186,7 @@ If a parameter of a function has a default value, please note what the default i
 If that default value is `None`, it can also be helpful to explain what happens when the parameter is `None`.
 If `**kwargs` is part of the function signature, link to the parent class(es) or method(s) so that the reader knows where to find the available parameters.
 
-For an example file with docstrings, see [the `circuit` module](https://github.com/amazon-braket/amazon-braket-sdk-python/blob/main/src/braket/circuits/circuit.py).
+For an example file with docstrings, see [the `gates` module](https://github.com/amazon-braket/autoqasm/blob/main/src/autoqasm/instructions/gates.py).
```

Comment:
The docstrings for `gates` are pretty minimal. Maybe we can use [`measurements` ](https://github.com/amazon-braket/autoqasm/blob/main/src/autoqasm/instructions/measurements.py) which also has a code example in module level doc and docs for returns.

Reply:
That's a good idea, thank you.